supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -359,6 +370,22 @@ Always spell out the full product name of the host, and do not capitalize the te
 
 *See also*: xref:syndesis[Syndesis], xref:fuse-online[Fuse Online]
 
+[discrete]
+[[red-hat-java]]
+==== image:images/no.png[no] Red Hat Java (noun)
+*Description*: Do not use _Red Hat Java_ to refer to the Red Hat distribution of the Open Java Development Kit (OpenJDK).
+
+Java is a registered trademark of Oracle and/or its affiliates.
+The Red Hat Java agreement with Oracle ended in 2023.
```

Comment:
@sangeetaraghu - I added this bit for context etc... but it could become obsolete if the situation changes in the future. WDYT? I can remove if overkill?
```suggestion
The Red Hat Java agreement with Oracle ended in 2023.
```

Reply:
@sangeetaraghu - Done. Thanks for reviewing :green_heart: